kuma/core/managers.py

```diff
@@ -241,6 +243,13 @@ def resolve_allowed_tags(model_obj, tags_curr, tags_new,
     return tags_out
 
 
+class IPBanManager(models.Manager):
+    def delete_old(self, days=30):
+        cutoff_date = date.today() - timedelta(days=days)
+        old_ip_bans = self.get_query_set().filter(created__lte=cutoff_date)
+        old_ip_bans.delete()
```

Comment:
I've taken to adding `deleted = models.DateTimeField(null=True,blank=True)` instead of actually deleting things, so there's some history. Then instead of actually deleting, you can make the model's/manager's `delete()` method just set the `deleted` attribute, and since you have a manager here anyway, maybe make an `active()` that does `super().filter(deleted__isnull=True)`.